api/src/main/resources/messages.properties

```diff
@@ -56,6 +56,13 @@ coreapps.task.mergeVisits.mergeSelectedVisits=Merge Selected Visits
 coreapps.task.mergeVisits.success=Visits merged successfully
 coreapps.task.mergeVisits.error=Failed to merge visits
 
+coreapps.task.deletePatient.label=Delete Patient
+coreapps.task.deletePatient.notAllowed=User does not have permission to delete patient
+coreapps.task.deletePatient.successMessage=Patient has been successfully deleted
+coreapps.task.deletePatient.deleteMessageEmpty = Reason cannot be empty
+coreapps.task.deletePatient.deletePatientSuccessful=Success! Patient has been deleted
+coreapps.task.deletePatient.deletePatientUnsuccessful=Error. Unable to delete patient
```

Comment:
I know I'm getting picky here, I think the informal convention is that this message normally is phrased as "Failed to delete the patient"